client/src/components/Searchbar/Searchbar.spec.js

```diff
@@ -0,0 +1,60 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import Searchbar from "./Searchbar";
+
+describe("Searchbar component", () => {
+  it("shows a dropdown to choose field of competence and input field for entering searchterms", () => {
+    const searchterm = {
+      competence: "",
+      buzzwords: "",
+    };
+    render(<Searchbar searchterm={searchterm} />);
+    expect(screen.getByRole("option", { name: /Choose field of competence/i }))
+      .toBeInTheDocument;
+    expect(
+      screen.getByRole("option", { name: /Architecture and Engineering/i })
+    ).toBeInTheDocument;
+    expect(screen.getByRole("option", { name: /Agriculture and Food/i }))
+      .toBeInTheDocument;
+    expect(screen.getByRole("option", { name: /Arts and Entertainment/i }))
+      .toBeInTheDocument;
+    expect(
+      screen.getByRole("option", {
+        name: /Business, Management and Administration/i,
+      })
+    ).toBeInTheDocument;
+    expect(screen.getByRole("option", { name: /Education and Training/i }))
+      .toBeInTheDocument;
+    expect(screen.getByRole("option", { name: /Education and Training/i }))
+      .toBeInTheDocument;
+    expect(screen.getByRole("option", { name: /Finance and Accounting/i }))
+      .toBeInTheDocument;
+    expect(screen.getByRole("option", { name: /Health and Medicine/i }))
+      .toBeInTheDocument;
+    expect(screen.getByRole("option", { name: /Law and Public Policy/i }))
+      .toBeInTheDocument;
+    expect(
+      screen.getByRole("option", {
+        name: /Sales, Marketing and Communications/i,
+      })
+    ).toBeInTheDocument;
+    expect(screen.getByRole("option", { name: /Science and Technology/i }))
+      .toBeInTheDocument;
+    expect(screen.getByPlaceholderText("Enter skill or buzzword..."))
+      .toBeInTheDocument;
```

Comment:
We checked out your project locally and all of these lines where red. `.toBeInTheDocument()`is a function, so you missed the function call. 
```suggestion
    expect(screen.getByRole("option", { name: /Choose field of competence/i }))
      .toBeInTheDocument();
    expect(
      screen.getByRole("option", { name: /Architecture and Engineering/i })
    ).toBeInTheDocument();
    expect(screen.getByRole("option", { name: /Agriculture and Food/i }))
      .toBeInTheDocument();
    expect(screen.getByRole("option", { name: /Arts and Entertainment/i }))
      .toBeInTheDocument();
    expect(
      screen.getByRole("option", {
        name: /Business, Management and Administration/i,
      })
    ).toBeInTheDocument();
    expect(screen.getByRole("option", { name: /Education and Training/i }))
      .toBeInTheDocument();
    expect(screen.getByRole("option", { name: /Education and Training/i }))
      .toBeInTheDocument();
    expect(screen.getByRole("option", { name: /Finance and Accounting/i }))
      .toBeInTheDocument();
    expect(screen.getByRole("option", { name: /Health and Medicine/i }))
      .toBeInTheDocument();
    expect(screen.getByRole("option", { name: /Law and Public Policy/i }))
      .toBeInTheDocument();
    expect(
      screen.getByRole("option", {
        name: /Sales, Marketing and Communications/i,
      })
    ).toBeInTheDocument();
    expect(screen.getByRole("option", { name: /Science and Technology/i }))
      .toBeInTheDocument();
    expect(screen.getByPlaceholderText("Enter skill or buzzword..."))
      .toBeInTheDocument();
```